package.json

```diff
@@ -40,6 +40,7 @@
     "tsd": "^0.23.0"
   },
   "dependencies": {
+    "fast-querystring": "^0.7.0",
```

Comment:
Is this tool stable? I don't like using <1.0.0 releases in production modules.

Reply:
It is stable and well-tested (I'm the author).

Reply:
Without it being at 1.0.0, I am not keen to make the change. Also,

https://github.com/nodejs/node/blob/5fad0b93667ffc6e4def52996b9529ac99b26319/test/parallel/test-querystring.js
vs
https://github.com/anonrig/fast-querystring/blob/b893058f36c2c1db615d6f238a70d696526ff7af/test/parse.test.ts

Shows that `querything` parsing is much more thoroughly tested in core.